src/normalizations/normalize-calendars.ts

```diff
@@ -38,9 +39,9 @@ export const normalizePartialCalendar = (folder: ZimbraFolder): any => ({
 	name: folder.name,
 	parent: folder.l,
 	rid: folder.rid,
+	n: folder.n,
 	owner: folder.owner, // It's specified only if It's not the current user
-	icon: folder.owner ? 'Share' : 'Calendar2',
 	zid: folder.zid,
 	acl: folder.acl,
-	isShared: !!folder.owner
+	isShared: !isNil(folder.owner) ? !!folder.owner : undefined
```

Comment:
why undefined ? it's a boolean so I think having a value is better to handle

Reply:
This is the normalization of a partial calendar used in the sync (if you watch the files there are two different normalizations). In the sync folders we only receive small parts of the calendar and not all of his properties. This small trick it is useful to avoid changing the "isShared" parameter improperly but instead updates it only when a folder is shared